src/es/components/molecules/tooltip/Tooltip.js

```diff
@@ -0,0 +1,190 @@
+// @ts-check
+import { Shadow } from '../../web-components-toolbox/src/es/components/prototypes/Shadow.js'
+
+/**
+* @export
+* @class Tooltip
+* @type {CustomElementConstructor}
+*/
+export default class Tooltip extends Shadow() {
+  constructor (options = {}, ...args) {
+    super({ importMetaUrl: import.meta.url, ...options }, ...args)
+  }
+
+  connectedCallback () {
+    if (this.shouldRenderCSS()) this.renderCSS()
+
+    this.text = this.getAttribute('text') || 'This is a fallback tooltip text!';
+
+    if (this.shouldRenderHTML()) this.renderHTML()
+
+    this.toggleTooltip();
+  }
+
+  disconnectedCallback () {
+    this.root.removeEventListener('click');
+  }
+
+    /**
+   * Toggle tooltip
+   */
+  toggleTooltip() {
+    const tooltip = this.root.querySelector('.tooltip');
+
+    this.root.addEventListener('click', () => {
+      tooltip.classList.toggle('tooltip-open');
+    });
+  }
+
+  /**
+   * evaluates if a render is necessary
+   *
+   * @return {boolean}
+   */
+  shouldRenderCSS () {
+    return !this.root.querySelector(`:host > style[_css], ${this.tagName} > style[_css]`)
+  }
+
+  /**
+   * evaluates if a render is necessary
+   *
+   * @return {boolean}
+   */
+  shouldRenderHTML () {
+    return !this.tooltip
```

Comment:
this.tooltip habe ich nicht gefunden. Auch keinen Getter. Ich würde da noch einen Getter auf dein Resultat von renderHTML machen. Dort schreibst du ja einen div mit Class Tooltip.

Der Getter würde grob so aussehen:

`get tooltip () {return this.root.querySelector(".tooltip")}`